Resources/views/CRUD/Association/list_collection.html.twig

```diff
@@ -0,0 +1,50 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{% extends admin.getTemplate('base_list_field') %}
+
+{% block field %}
+    {% set route_name = field_description.options.route.name %}
+    {% set route_role = route_name|upper %}
```

Comment:
That's how it's [currently done](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/Resources/views/CRUD/list_orm_many_to_many.html.twig#L14).
You confirm it should be changed ?